src/host/buildvm_asm.c

```diff
@@ -89,8 +123,12 @@ void emit_asm(BuildCtx *ctx) {
     while (rel < ctx->nreloc && ctx->reloc[rel].ofs <= next) {
```

Comment:
[This line](https://github.com/igelhaus/luavela/blob/macosx-build-rc/src/host/buildvm_asm.c#L117) is emitted only for elfasm, considering [upstream](https://github.com/LuaJIT/LuaJIT/blob/v2.0/src/host/buildvm_asm.c#L227). @igelhaus, if everything is fine now, please drop a comment regarding this difference with upstream.

P.S. gh review is too hard to me, sorry.

Reply:
This line simply emits a label, which does no harm (or at least I hope so). But you are right, let's stay aligned.